files/riot/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "riotjs",
+  "repo": "muut/riotjs",
+  "files": {
+    "include": ["riot.js", "riot.min.js"]
```

Comment:
Yea, these files weren't there in previous versions. They're already in `master` however, and will be included in all upcoming releases from now on.